js/dataframe/shared/Table.svelte

```diff
@@ -956,7 +956,6 @@
 		top: 0;
 		left: 0;
 		z-index: var(--layer-1);
-		box-shadow: var(--shadow-drop);
```

Comment:
my reasoning was that this is tied directly to --shadow-drop, not some --table-header-shadow-drop specifically for this usecase, so this would get enabled automatically if you use shadow drop anywhere. For that reason I removed it